dpp/src/tests/utils/utils.rs

```diff
@@ -98,11 +96,49 @@ pub fn get_data_from_file(file_path: &str) -> Result<String> {
     Ok(d)
 }
 
-pub fn decode_hex(s: &str) -> Result<Vec<u8>, ParseIntError> {
-    (0..s.len())
-        .step_by(2)
-        .map(|i| u8::from_str_radix(&s[i..i + 2], 16))
-        .collect()
+pub trait SerdeTestExtension {
```

Comment:
good idea!, If this is used in tests, probably we should use `#[cfg...` as it could be auto-imported in production code.

Reply:
Done